config.js

```diff
@@ -596,6 +596,12 @@ config.QUOTA_MAX_OBJECTS = Number.MAX_SAFE_INTEGER;
 //////////////////////////
 config.STS_DEFAULT_SESSION_TOKEN_EXPIRY_MS = 60 * 60 * 1000; // 1 hour
 
+///////////////////////////////////////////
+//      PostgreSQL client pool size      //
+///////////////////////////////////////////
+config.POSTGRES_MAX_CLIENTS = 80
+config.POSTGRES_DEFAULT_CLIENTS = 10
```

Comment:
I suggest having one `POSTGRES_MAX_CLIENTS` config, and in postgres_client, just use it. It will make it easier to override without trying to remember which one is used in each case

```suggestion
config.POSTGRES_MAX_CLIENTS = (process.env.LOCAL_MD_SERVER === 'true') ? 80 : 10;
```